demos/JWST/S3_miri_lrs_template.ecf

```diff
@@ -14,9 +14,13 @@ src_pos_type    gaussian    # Determine source position when not given in header
 record_ypos     True        # Option to record the y position and width for each integration (only records if src_pos_type is gaussian)
 dqmask          True        # Mask pixels with an odd entry in the DQ array
 
+# Outlier rejection along time axis
+ff_outlier      False       # Set True to use full frame; set False to use only background region
+ff_thresh       [5,5]       # Full Frame double-iteration X-sigma threshold for outlier rejection along time axis
```

Comment:
These two new options need to be documented in the ECF template in docs/media/ as well as the docs/source/ecf.rst file. You should also make it clear in those docs that bg_thresh isn't used if ff_outlier is set to True and ff_thresh is used instead. Actually, since the two thresholds have the same purpose and only one of them is ever used at the same time, why have two different names? Why not just use meta.bg_thresh inside the new ff_outlier algorithm?

Reply:
Good point.  I removed ``ff_thresh`` completely and added descriptions to the ecf.rst file.